integrations/ingress-nginx/documentation.yaml

```diff
@@ -0,0 +1,90 @@
+exporter_type: included
+app_name_short: Ingress NGINX
+app_name: {{app_name_short}}
+app_site_name: Ingress NGINX
+app_site_url: https://kubernetes.github.io/ingress-nginx
+exporter_name: the Ingress NGINX exporter
+exporter_pkg_name: ingress-nginx
+exporter_repo_url: https://kubernetes.github.io/ingress-nginx/user-guide/monitoring/
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. You can run the following checks to verify
+  that {{app_name_short}} is exposing Prometheus-format metrics.
+
+  To determine if {{exporter_name}} is exposing metrics, run the following command:
```

Comment:
You could make this (lines 11-14) shorter by just saying:

...separately. To verify that {{app_name_short}} is exposing metrics, run the following command: